source/shared/cpp/ObjectModel/UnknownElement.cpp

```diff
@@ -0,0 +1,49 @@
+#include <iomanip>
+#include <regex>
+#include <iostream>
+#include <codecvt>
+#include "UnknownElement.h"
+#include "DateTimePreparser.h"
+#include "ParseUtil.h"
+
+using namespace AdaptiveCards;
+
+UnknownElement::UnknownElement() :
+    BaseCardElement(CardElementType::Unknown)
+{
+}
+
+UnknownElement::UnknownElement(
+    Json::Value const &paylaod) :
+    BaseCardElement(CardElementType::Unknown)
+{
+}
+
+Json::Value UnknownElement::SerializeToJsonValue()
+{
+    Json::Value root = GetJsonPayload();
+    root[AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::Type)] = CardElementTypeToString(CardElementType::Unknown); 
```

Comment:
>root[AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::Type)] = CardElementTypeToString(CardElementType::Unknown);  [](start = 4, length = 118)

You need the other base card element properties as well. Call BaseCardElement::SerializeToJsonValue

Reply:
Is there a way to append two Json::Values, otherwise i will just add those properties by name.

---
In reply to: [172373459](https://github.com/Microsoft/AdaptiveCards/pull/1208#discussion_r172373459) [](ancestors = 172373459)